docs/quickstart/installation-guide-for-beginners.mdx

```diff
@@ -0,0 +1,257 @@
+
+    g. Wait until you see a green dot next to "Docker Desktop is running" in the Docker Dashboard.
+
+    #### 3. Install KinD
+
+    KinD (Kubernetes in Docker) helps set up a test environment for Odigos.
+
+    a. Go back to your Terminal window (or open a new one if you closed it).
+
+    b. Copy and paste the following command, then press `Enter`:
+
+    ```
+    brew install kind
+    ```
+
+    c. Wait for the installation to complete. You'll know it's done when you see the command prompt again.
+
+    ### Installing Odigos
+
+    #### 1. Set up a test environment
+
+    a. In the Terminal, copy and paste the following command and press `Enter`:
+
+    ```
+    kind delete cluster --name odigos-demo
+    ```
+
+    This removes any existing test environment with the same name. If this is your first time, you'll see a message saying the cluster doesn't exist, which is fine.
+
+    b. Now, copy and paste this command and press `Enter`:
+
+    ```
+    kind create cluster --name odigos-demo
+    ```
+
+    This creates a new test environment.
+
+    c. You'll see a lot of text appear in the Terminal. This is normal. Wait until you see a message that says "Your Kubernetes control-plane has initialized successfully!" and the command prompt appears again.
+
+    #### 2. Install the Odigos CLI tool
+
+    The Odigos CLI (Command Line Interface) tool allows you to manage Odigos from the Terminal.
+
+    a. In the Terminal, copy and paste the following command and press `Enter`:
+
+    ```
+    brew install odigos-io/homebrew-odigos-cli/odigos
+    ```
+
+    b. Wait for the installation to complete. You'll know it's done when you see the command prompt again.
+
+    #### 3. Install Odigos in your test environment
+
+    a. In the Terminal, copy and paste the following command and press `Enter`:
+
+    ```
+    odigos install
+    ```
+
+    b. This step may take several minutes. You'll see a lot of text appear in the Terminal – this is normal.
+
+    c. You'll know it's done when you see a message saying "SUCCESS: Odigos installed." and the command prompt appears again.
+
+    #### 4. Open the Odigos control panel
+
+    a. In the Terminal, copy and paste the following command and press `Enter`:
+
+    ```
+    odigos ui
+    ```
+
+    After you run this command, the Terminal will look like it's frozen. This is normal – it's running a program in the background.
+
+    b. Open your web browser (like Safari or Chrome).
+
+    c. In the address bar at the top of the browser window, type exactly this:
+
+    ```
+    http://localhost:3000
+    ```
+
+    Then press `Enter`.
+
+    d. The Odigos control panel should now open in your browser. It might take a few seconds to load.
+
+    #### 5. Set up Odigos
+
+    a. In the Odigos control panel, you'll see a page titled "Select Sources". Sources are the things Odigos can monitor.
+
+    b. Click the checkbox next to each Source you want to monitor. If you're not sure, you can select all of them.
```

Comment:
This guide skipped the part about installing the demo application, thus there will be no sources at this point